<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BIOSSettingsWidget</class>
 <widget class="QWidget" name="BIOSSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>BIOS Directory</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>PCSX2 will search for BIOS images in this directory.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="buddy">
         <cstring>searchDirectory</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="searchDirectory"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseSearchDirectory">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetSearchDirectory">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>BIOS Selection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="openSearchDirectory">
          <property name="text">
           <string>Open BIOS Folder...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refresh">
          <property name="text">
           <string>Refresh List</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QTreeWidget" name="fileList">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <attribute name="headerMinimumSectionSize">
         <number>250</number>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>250</number>
        </attribute>
        <column>
         <property name="text">
          <string>Filename</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Version</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options and Patches</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="fastBoot">
          <property name="text">
           <string>Fast Boot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fastBootFastForward">
          <property name="text">
           <string>Fast Forward Boot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>searchDirectory</tabstop>
  <tabstop>browseSearchDirectory</tabstop>
  <tabstop>resetSearchDirectory</tabstop>
  <tabstop>fileList</tabstop>
  <tabstop>openSearchDirectory</tabstop>
  <tabstop>refresh</tabstop>
  <tabstop>fastBoot</tabstop>
  <tabstop>fastBootFastForward</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
